assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -1,43 +1,178 @@
 //default color scheme
 :root {
 	--color-primary: #{$muriel-blue-500};
-	--color-primary-light: #{$muriel-blue-300};
-	--color-primary-dark: #{$muriel-blue-700};
 	--color-primary-rgb: #{hex-to-rgb( $muriel-blue-500 )};
+	--color-primary-dark: #{$muriel-blue-700};
+	--color-primary-dark-rgb: #{hex-to-rgb( $muriel-blue-700 )};
+	--color-primary-light: #{$muriel-blue-300};
+	--color-primary-light-rgb: #{hex-to-rgb( $muriel-blue-300 )};
+	--color-primary-0: #{$muriel-blue-0};
+	--color-primary-0-rgb: #{hex-to-rgb( $muriel-blue-0 )};
+	--color-primary-50: #{$muriel-blue-50};
+	--color-primary-50-rgb: #{hex-to-rgb( $muriel-blue-50 )};
+	--color-primary-100: #{$muriel-blue-100};
+	--color-primary-100-rgb: #{hex-to-rgb( $muriel-blue-100 )};
+	--color-primary-200: #{$muriel-blue-200};
+	--color-primary-200-rgb: #{hex-to-rgb( $muriel-blue-200 )};
+	--color-primary-300: #{$muriel-blue-300};
+	--color-primary-300-rgb: #{hex-to-rgb( $muriel-blue-300 )};
+	--color-primary-400: #{$muriel-blue-400};
+	--color-primary-400-rgb: #{hex-to-rgb( $muriel-blue-400 )};
+	--color-primary-500: #{$muriel-blue-500};
+	--color-primary-500-rgb: #{hex-to-rgb( $muriel-blue-500 )};
```

Comment:
I think the former two are redundant, aren't they?

Reply:
They are for this theme, but may not be for other themes.